native/src/components/MapView.tsx

```diff
@@ -16,22 +16,29 @@ import {
   animationDuration,
 } from 'api-client'
 
+import { GPSFixed, GPSNotFixed, GPSOff } from '../assets'
 import { clusterCountLayer, clusterLayer, markerLayer } from '../constants/layers'
 import { midSnapPointPercentage } from '../routes/Pois'
 import MapAttribution from './MapsAttribution'
+import Icon from './base/Icon'
+import IconButton from './base/IconButton'
 
 const MapContainer = styled.View`
   flex: 1;
   flex-direction: row;
   justify-content: center;
 `
+
 const StyledMap = styled(MapLibreGL.MapView)`
   width: 100%;
 `
 
-const StyledFAB = styled(FAB)<{ position: number | string }>`
-  align-items: flex-end;
+const StyledIcon = styled(IconButton)<{ position: number | string }>`
+  position: absolute;
+  right: 0;
```

Comment:
Looks like this might break on RTL layouts

Reply:
Should be fine, we don't move the icon at the moment either.

Reply:
Actually react-native automatically moves this to the other side for RTL layouts so this is fine.

Reply:
Huh. Cool